st2web/files/st2.conf-http.patch

```diff
@@ -28,37 +21,33 @@
 -
 -server {
 -  listen       *:443 ssl;
+-  server_tokens off;
+-
+-  if ($request_method !~ ^(GET|HEAD|POST|PUT|DELETE|OPTIONS)$ ) {
+-       return 405;
+-  }
 -
 -  ssl_certificate           /etc/ssl/st2/st2.crt;
 -  ssl_certificate_key       /etc/ssl/st2/st2.key;
 -  ssl_session_cache         shared:SSL:10m;
 -  ssl_session_timeout       5m;
--  ssl_protocols             TLSv1 TLSv1.1 TLSv1.2;
+-  ssl_protocols             TLSv1.2;
 -  ssl_ciphers               EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH:ECDHE-RSA-AES128-GCM-SHA384:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA128:DHE-RSA-AES128-GCM-SHA384:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES128-GCM-SHA128:ECDHE-RSA-AES128-SHA384:ECDHE-RSA-AES128-SHA128:ECDHE-RSA-AES128-SHA:ECDHE-RSA-AES128-SHA:DHE-RSA-AES128-SHA128:DHE-RSA-AES128-SHA128:DHE-RSA-AES128-SHA:DHE-RSA-AES128-SHA:AES128-GCM-SHA384:AES128-GCM-SHA128:AES128-SHA128:AES128-SHA128:AES128-SHA:AES128-SHA:HIGH:!aNULL:!eNULL:!EXPORT:!DES:!MD5:!PSK:!RC4;
 -  ssl_prefer_server_ciphers on;
 -
    index  index.html;
  
--  access_log            /var/log/nginx/ssl-st2webui.access.log combined;
--  error_log             /var/log/nginx/ssl-st2webui.error.log;
--
--  add_header              Front-End-Https on;
--  add_header              X-Content-Type-Options nosniff;
-+  access_log /proc/self/fd/1 combined;
-+  error_log stderr;
```

Comment:
Looks like this part is missing

Reply:
Which part?

I think you should rook at the raw files from master and this PR since diffing two diffs makes it very hard to review :)

Reply:
EDIT: Nvm, I see the log stuff is missing, yeah.

Reply:
I think it should be better now:

 https://github.com/StackStorm/st2-dockerfiles/blob/master/st2web/files/st2.conf-http.patch
https://github.com/StackStorm/st2-dockerfiles/blob/fix_build/st2web/files/st2.conf-http.patch

Reply:
We also need more robust approach going forward since this diffing approach is a pain and not robust.